test-suite/generated-src/java/com/dropbox/djinni/test/AssortedPrimitives.java

```diff
@@ -194,4 +194,111 @@ public String toString() {
         "}";
     }
 
+
+    public static final android.os.Parcelable.Creator<AssortedPrimitives> CREATOR
+        = new android.os.Parcelable.Creator<AssortedPrimitives>()
+    {
+        @Override
+        public AssortedPrimitives createFromParcel(android.os.Parcel in)
+        {
+            return new AssortedPrimitives(in);
+        }
+
+        @Override
+        public AssortedPrimitives[] newArray(int size)
+        {
+            return new AssortedPrimitives[size];
+        }
+    };
+
+    public AssortedPrimitives(android.os.Parcel in)
+    {
+        this.mB = in.readByte() != 0;
+        this.mEight = in.readByte();
+        this.mSixteen = (short)in.readInt();
+        this.mThirtytwo = in.readInt();
+        this.mSixtyfour = in.readLong();
+        this.mFthirtytwo = in.readFloat();
+        this.mFsixtyfour = in.readDouble();
+        if (in.readByte() == 0)
```

Comment:
Please generate braces around ifs, even in generated code.  Missing them makes me nervous about unintended consequences when people insert lines later.